android/app/build.gradle

```diff
@@ -81,12 +81,13 @@ project.ext.react = [
 ]
 
 project.ext.envConfigFiles = [
-        devDebug: ".env.dev",
-        prodRelease: ".env.prod",
-        stagingRelease: ".env.staging"
+    devDebug: ".env.dev",
+    prodRelease: ".env.prod",
+    stagingRelease: ".env.staging"
```

Comment:
Don't mind this configuration for now because it's not even working for the time being. @ruimonteiro93 is going to come up with a solution to setup the environment config soon enough.